src/components/inputs/EthAddressInput.tsx

```diff
@@ -76,6 +78,10 @@ export function EthAddressInput({
     }
   }
 
+  useEffect(() => {
```

Comment:
added this pattern to ensure the input does ENS processing on any initial value. previously the ENS name lookup would only happen when the input changed